graphql/tests/repoSettings/results.json

```diff
@@ -251,16 +251,17 @@
         "data": null,
         "errors": [
           {
-            "message": "repo doesn't exist",
+            "message": "user testuser does not have permission to access settings for the project evergreen",
```

Comment:
This error message is a bit less correct than before because of how the directive works. I didn't think it was worth performing an additional DB lookup in the projectRef and repoRef collections to present a correct error message, but interested in your thoughts.